src/components/createField.jsx

```diff
@@ -84,8 +84,8 @@ export default function connectField(options) {
       }
 
       static contextTypes = {
-        form: PropTypes.object.isRequired,
```

Comment:
Removed isRequired for form and field as, if the component is not wrapped with Form, it will throw PropTypes error before custom error message is shown.